web/themes/custom/server_theme/templates/server-theme-person-card.html.twig

```diff
@@ -0,0 +1,53 @@
+<div
+	class="flex flex-col items-start p-0 relative bg-white shadow rounded-lg border-gray-300 relative">
+	{# Leading content #}
+	<div
+		class="flex flex-col items-center p-8 gap-6 w-full flex-none order-0 self-stretch grow-0">
+		{# Avatar #}
+		<figure class="flex-none order-0 grow-0 flex flex-col justify-center items-center p-0 rounded-full overflow-hidden">
+			{{ image }}
+		</figure>
+		{# Text #}
+		<div
+			class="flex-none order-1 grow-0 self-stretch flex flex-col justify-center items-center p-0 g-2">
+			{# Title #}
+			<div class="w-full not-italic font-medium text-sm leading-5 text-center text-gray-900 flex-none order-0 self-stretch grow-0">
+				{{ title }}
+			</div>
+			{# Detail and badge #}
+			<div
+				class="flex-none order-1 grow-0 self-stretch flex flex-col items-center p-0 gap-y-2">
+				{# Detail #}
+				<div class="flex-none order-0 self-stretch grow-0 not-italic font-normal text-sm leading-5 text-center text-gray-500">
+					{{ detail }}
+				</div>
+				{# Badge #}
+				<div class="flex-none order-1 grow-0 flex flex-row justify-center items-center pt-1 pr-2 pb-1 pl-2 bg-green-100 rounded-2xl">
+					<div class="flex-none order-0 grow-0 not-italic font-medium text-xs leading-4 text-center text-green-800">{{ badge }}</div>
+				</div>
+			</div>
+		</div>
+	</div>
+	{# Divider #}
+	<div class="h-px flex-none order-1 grow-0 self-stretch bg-gray-200"></div>
```

Comment:
Why did you go with `order-x` here? Is there a different way?

Reply:
Well this is embarassing, in my local it looks like these are spaces only. Need to check if twig formatter doing something crazy or some other mac encoding issue